config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # studios 
+  get "/studios", to: "studios#index" 
+
+  # movies 
+  get "/movies/:id", to: "movies#show"
+  
```

Comment:
Given the form route refactor, you would need to add a POST route that directs to a joins controller #create action:
`post "/movies/:id/actors", to: "movie_actors#create"`